xml/operations-objectstorage-swift_container_sync.xml

```diff
@@ -328,6 +328,12 @@ Additional middleware: container_sync
 key = lQ8JjuZfO
```

Comment:
Should we change this key for 'secret123'  too ?

Reply:
No, that one is fine. That's the container-sync realms file. The one Ops set up. so in that case it _is_ the realm key. The one a normal user shouldn't know about.

the sectret123 is the user generated key they add to both containers they want to sync between.

A HMAC is generated on the object, realm key, user key, operation etc, so the other cluster can generate a matching HMAC to see if it's allowed.